peru/resources/plugins/git/git_plugin.py

```diff
@@ -167,10 +168,33 @@ def plugin_reup(url, reup):
         print('rev:', output.strip(), file=out_file)
 
 
+def git_default_branch(url) -> str:
+    """
+    With a list of common default branches names, it checks against
+    the available branch list if this possible default branch exists.
+    By default git used to take master as default branch, but newer versions
+    tend to use main, even trunk or development can be found.
+
+    Args:
+        url (str): url from the target repository to be checked.
+    Returns:
+        str: returns a possible match for the git default branch.
+    """
+    repo_path = clone_if_needed(url)
+    default_branches_list = ['master', 'main', 'trunk', 'development']
```

Comment:
My personal preference is to just use `master` and `main` here. I think folks with more customized setups should use the explicit `rev` or `reup` fields in their `peru.yaml`.

Reply:
Agreed.